### PR TITLE
LPS-197622 Create Test for DDLRecordVersion TableReferenceDefinition

### PR DESCRIPTION
**Pull request by gislayne-vitorino**
This is an update for: [LPS-197622](https://liferay.atlassian.net/browse/LPS-197622)

### CODE REVIEW

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:stop

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:test:publications

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Comment by gislayne-vitorino** [comment]
ci:test:publications

File: modules/apps/dynamic-data-lists/dynamic-data-lists-test/src/testIntegration/java/com/liferay/dynamic/data/lists/change/tracking/test/DDLRecordVersionTableReferenceDefinitionTest.java [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		DDLRecordSetTestHelper ddlRecordSetTestHelper =
+			new DDLRecordSetTestHelper(group);
+		DDMStructure ddmStructure = DDMStructureTestUtil.addStructure(
+			group.getGroupId(), DDLRecordSet.class.getName());
+
+		DDLRecordSet ddlRecordSet = ddlRecordSetTestHelper.addRecordSet(
+			ddmStructure);
+
+		DDLRecordTestHelper recordTestHelper = new DDLRecordTestHelper(
+			group, ddlRecordSet);
```
Comment (pei-jung) [inline modules/apps/dynamic-data-lists/dynamic-data-lists-test/src/testIntegration/java/com/liferay/dynamic/data/lists/change/tracking/test/DDLRecordVersionTableReferenceDefinitionTest.java:58]: Can you rename this to `ddlRecordTestHelper`?

**Review by pei-jung — CHANGES_REQUESTED** [review]

**Comment by gislayne-vitorino** [comment]
ci:test:sf

**Review by pei-jung — APPROVED** [review]

**Comment by pei-jung** [comment]
ci:test:relevant

**Comment by pei-jung** [comment]
ci:forward

**Pull request closed without merging**